packages/RNTester/js/examples/Modal/builder.js

```diff
@@ -0,0 +1,271 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict-local
+ */
+
+import React from 'react';
+import {
+  Modal,
+  Switch,
+  Text,
+  TouchableHighlight,
+  View,
+  StyleSheet,
+  Platform,
+} from 'react-native';
+import {Picker} from '@react-native-community/picker';
+
+const Item = Picker.Item;
+
+class Button extends React.Component {
```

Comment:
@YashKumarVerma , you can try using the hooks and functional components here.

Reply:
Similarly for other class components.

Reply:
yep, on it. actually i left this file untouched.
will be pushing the new code by tomorrow morning now.